src/submission/submission-entities.ts

```diff
@@ -365,3 +392,41 @@ export interface ClinicalSubmissionRecordsByDonorIdMap {
 export interface SubmittedClinicalRecordsMap {
   [type: string]: SubmittedClinicalRecord[];
 }
+
+export const ClinicalEntityToEnumFieldsMap: Record<ClinicalEntitySchemaNames, string[]> = {
+  [ClinicalEntitySchemaNames.REGISTRATION]: Object.values(SampleRegistrationFieldsEnum),
+  [ClinicalEntitySchemaNames.DONOR]: Object.values(DonorFieldsEnum),
+  [ClinicalEntitySchemaNames.SPECIMEN]: Object.values(SpecimenFieldsEnum),
+  [ClinicalEntitySchemaNames.PRIMARY_DIAGNOSIS]: Object.values(PrimaryDiagnosisFieldsEnum),
+  [ClinicalEntitySchemaNames.FOLLOW_UP]: Object.values(FollowupFieldsEnum),
+  [ClinicalEntitySchemaNames.TREATMENT]: Object.values(TreatmentFieldsEnum),
+  [ClinicalEntitySchemaNames.CHEMOTHERAPY]: Object.values(ChemotherapyFieldsEnum),
+  [ClinicalEntitySchemaNames.RADIATION]: Object.values(RadiationFieldsEnum),
+  [ClinicalEntitySchemaNames.HORMONE_THERAPY]: Object.values(HormoneTherapyFieldsEnum),
+};
+
+export const TreatmentTypeValuesMappedByTherapy: Record<ClinicalTherapyType, string[]> = {
+  [ClinicalEntitySchemaNames.CHEMOTHERAPY]: [
+    'Chemotherapy',
+    'Combined chemo+immunotherapy',
+    'Combined chemo+radiation therapy',
+    'Combined chemo-radiotherapy and surgery',
+  ],
+  [ClinicalEntitySchemaNames.RADIATION]: [
+    'Radiation therapy',
+    'Combined chemo+radiation therapy',
+    'Photodynamic therapy',
+  ],
+  [ClinicalEntitySchemaNames.HORMONE_THERAPY]: ['Hormonal therapy'],
+};
+
+export const DonorVitalStatusTimeSensitiveValues = ['Deceased'];
```

Comment:
I would put this as an enum / constant object and use the key in the code

Reply:
I did thing of that but what if there are multiple values, e.g. `Dead` gets added so we are looking to match `Dead` or `Deceased`